templates/step_function/consignment_export_definition.json.tpl

```diff
@@ -1,19 +1,136 @@
 {
   "Comment": "A state machine to run the Fargate task to export the consignment",
-  "StartAt": "Run ECS task",
+  "StartAt": "Parallel",
```

Comment:
You'll need to incorporate Thanh's changes to implement retry logic on ECS execution failures into the separate task definitions: https://github.com/nationalarchives/tdr-terraform-environments/pull/452

Reply:
I did. They both have the `"ECS.AmazonECSException"` in the catch block and use the `max_attempts` variable. You might have written this before I did that now I think about it.